analysis_runner/dataproc.py

```diff
@@ -47,6 +48,12 @@ def hail_dataproc_job(
 
     cluster_name = f'dataproc-{uuid.uuid4().hex}'
 
+    # Add default labels to passed labels.
+    labels.update({'compute_category': 'dataproc'})
```

Comment:
Does this work when `labels` is `None`?

Reply:
Is the plan to also add this a default `compute_category` to Cromwell workflows?

Reply:
Yes to both :)

Reply:
Are you sure? Wouldn't this call `None.update`?

Reply:
ah great point! I conflated `None` and `{}` in my head.